src/site/_data/contributors.json

```
@@ -1400,5 +1400,19 @@
     "twitter": "mgechev",
     "github": "mgechev",
     "glitch": "mgechev"
+  },
+  "mohamedzamakhan": {
+    "name": {
+      "given": "Zama Khan",
+      "family": "Mohammed"
+    },
+    "org": {
+      "name": "Tekzenit Inc.",
+      "unit": "Software Engineer"
```

```suggestion
      "unit": "Software Engineering"
```
The unit is the department or division you're in, not your job title. Please fix if this isn't the right name.